src/structures/IntegrationApplication.js

```diff
@@ -15,6 +15,48 @@ class IntegrationApplication extends Application {
      * @type {?User}
      */
     this.bot = data.bot ? this.client.users.add(data.bot) : this.bot ?? null;
+
+    /**
+     * The url of the app's terms of service
+     * @type {?string}
+     */
+    this.termsOfServiceURL = data.terms_of_service_url ?? this.terms_of_service_url ?? null;
+
+    /**
+     * The url of the app's privacy policy
+     * @type {?string}
+     */
+    this.privacyPolicyURL = data.privacy_policy_url ?? this.privacyPolicyURL ?? null;
+
+    /**
+     * The Array of RPC origin urls
+     * @type {string[]}
+     */
+    this.rpcOrigins = data.rpc_origins ?? this.rpcOrigins ?? [];
+
+    /**
+     * The application summary
+     * @type {?string}
+     */
+    this.summary = data.summary ?? this.summary ?? null;
+
+    /**
+     * Whether the application has been associated with a webhook
+     * @type {?boolean}
+     */
+    this.hook = data.hook ?? this.hook ?? null;
+
+    /**
+     * The hash of the application's cover image
+     * @type {?string}
+     */
+    this.cover = data.cover_image ?? this.cover ?? null;
+
+    /**
+     * The verification key of the application
```

Comment:
```suggestion
     * The hex-encoded key for verification in interactions and the GameSDK's GetTicket
```
from https://discord.com/developers/docs/topics/oauth2#application-application-structure